src/coreclr/scripts/superpmi.py

```diff
@@ -311,6 +311,7 @@
 asm_diff_parser.add_argument("-base_jit_option", action="append", help="Option to pass to the baseline JIT. Format is key=value, where key is the option name without leading COMPlus_...")
 asm_diff_parser.add_argument("-diff_jit_option", action="append", help="Option to pass to the diff JIT. Format is key=value, where key is the option name without leading COMPlus_...")
 asm_diff_parser.add_argument("-tag", help="Specify a word to add to the directory name where the asm diffs will be placed")
+asm_diff_parser.add_argument("-metrics", action="append", help="Metrics option to pass to jit-analyze. Can be specified multiple times, or pass comma-separated values.")
```

Comment:
It would have been good to print what metrics are allowed, but it is not possible to get that information from `jit-analyze` . How does the error looks like if we pass wrong metric?

Reply:
Looks like:
```
Invoking: c:\gh\jitutils\bin\jit-analyze.bat --md c:\spmi\asm.benchmarks.run.windows.x64.checked.19\summary.md -r --base c:\spmi\asm.benchmarks.run.windows.x64.checked.19\base --diff c:\spmi\asm.benchmarks.run.windows.x64.checked.19\diff --metrics foobar,CodeSize,PerfScore
error: Unknown metric 'foobar'. Available metrics: CodeSize, PrologSize, PerfScore, InstrCount, AllocSize, ExtraAllocBytes, DebugClauseCount, DebugVarCount, SpillCount, SpillWeight, ResolutionCount, ResolutionWeight
Asm diffs summary:
Traceback (most recent call last):
  File "c:\gh\runtime\src\coreclr\scripts\superpmi.py", line 3999, in <module>
    sys.exit(main(args))
  File "c:\gh\runtime\src\coreclr\scripts\superpmi.py", line 3921, in main
    success = asm_diffs.replay_with_asm_diffs()
  File "c:\gh\runtime\src\coreclr\scripts\superpmi.py", line 2026, in replay_with_asm_diffs
    with open(summary_file, "r") as read_fh:
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\spmi\\asm.benchmarks.run.windows.x64.checked.19\\summary.md'
Error
```